tracker/tests/conftest.py

```diff
@@ -0,0 +1,37 @@
+"""Test configuration and fixtures for eval-track."""
+
+import os
+from typing import Generator
+
+import pytest
+from fastapi.testclient import TestClient
+
+from tracker.tracker.main import app
+from tracker.tracker.settings import Settings, settings
```

Comment:
```suggestion
from tracker.main import app
from tracker.settings import Settings, settings
```

other files have the same pattern mistakes, please fix as follows